packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -108,7 +108,7 @@ export function DatePicker({
 function useOnClickOutside<T extends HTMLElement>(onClickOutside: () => void): MutableRefObject<T | null> {
```

Comment:
This should be renamed since the functionality has been changed.

Reply:
Can we use the clickoutside hoc we have? And create another one for focus-loss perhaps?